scripts/cfos/tests/README.md

```diff
@@ -1,18 +1,24 @@
 ## Test schema file setup ##
-You can either:
-- replace the contents of 'scripts/cfos/dataset_tables_schema.json' file with the contents of the 'scripts/cfos/tests/test_schema.json' file temporarily OR
-- change the code in make_dataset_data_tables.py so that schema_json points to the test schema file instead of the real one
+You can run the code using the test schema by adding the parameter "-t True" to run in testing mode (examples shown below in How to Run section)
 
 ## How to Run ##
 ### test fail ### 
-python3 make_dataset_data_tables.py -d schema1 -x tests/CFoS_Template_Test_Schema1_Fail.xlsx -p broad-cfos-data-platform1 -w cFOS_automation_testing
+python3 make_dataset_data_tables.py -d schema1 -x tests/CFoS_Template_Test_Schema1_Fail.xlsx -p broad-cfos-data-platform1 -w cFOS_automation_testing -t True
 
 ### test pass ###
-python3 make_dataset_data_tables.py -d schema1 -x tests/CFoS_Template_Test_Schema1_Pass.xlsx -p broad-cfos-data-platform1 -w cFOS_automation_testing
+python3 make_dataset_data_tables.py -d schema1 -x tests/CFoS_Template_Test_Schema1_Pass.xlsx -p broad-cfos-data-platform1 -w cFOS_automation_testing -t True
 
 
 ## Validating outcomes ##
 Check the validation_erros.csv and make sure expected outcomes are met. 
 General expected outcomes are explained above the column ID in the Fail.xlsx file. 
 
+The code can fail in the following ways:
+- **primary key column contains non-unique values**: 'validation_error':{row: 6, column: "id_unique"}: "X1234B" contains values that are not unique
```

Comment:
is "X1234B" a value or a column name? I think it's a value (the message suggests that the column name is "id_unique") - in which case I find this wording confusing - how can a single value contain values that are not unique? it points to row 6, but again wouldn't this suggest that there are multiple rows containing duplicate values?

Reply:
This is the value. It is indicating which values in the column are not unique. In the actual version it'd write that for every non-unique value in the column, but I don't think that's necessary for this example. 

This is the output for the pandas_schema is_distinct validation.